src/plugins/local-llm-rename/llama.ts

```diff
@@ -13,13 +19,16 @@ const IS_CI = process.env["CI"] === "true";
 export async function llama(opts: {
   seed?: number;
   model: string;
-  disableGPU?: boolean;
+  disableGpu?: boolean;
 }): Promise<Prompt> {
-  const llama = await getLlama();
-  const model = await llama.loadModel({
+  const disableGpu = opts.disableGpu ?? IS_CI;
+  const llama = await getLlama({ gpu: disableGpu ? false : "auto" });
+  const modelOpts: LlamaModelOptions = {
     modelPath: getModelPath(opts?.model),
-    gpuLayers: (opts?.disableGPU ?? IS_CI) ? 0 : undefined
-  });
+    gpuLayers: disableGpu ? 0 : undefined
```

Comment:
Curious, if GPU is disabled in the call to `getLlama`, do you actually need to set `gpuLayers` to `0` here? Or is this just leftover from your previous attempted approach to disable GPU?